docs/usage/kotlin-script.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Please move this content into `lib/modules/manager/kotlin-script/readme.md`

Reply:
After this change, unit tests fail with

```
 ● modules/manager/metadata › kotlin-script has readme with no h1 or h2
    expected true to be false or Boolean(false)
      31 |     expect(
      32 |       res.some((line) => line.startsWith('# ') || line.startsWith('## '))
    > 33 |     ).toBeFalse();
         |       ^
      34 |   });
      35 | });
      36 |
      at lib/modules/manager/metadata.spec.ts:33:7
```

Does it mean I shouldn't use headers in this manager-related REDME...?

Reply:
@rarkins friendly ping about this :arrow_up:

Reply:
I just removed the only header there.